crates/router/src/core/webhooks.rs

```diff
@@ -652,9 +652,14 @@ pub async fn webhooks_core<W: api::OutgoingWebhookType>(
     req: &actix_web::HttpRequest,
     merchant_account: domain::MerchantAccount,
     key_store: domain::MerchantKeyStore,
-    connector_name: &str,
+    connector_label: &str,
     body: actix_web::web::Bytes,
 ) -> RouterResponse<serde_json::Value> {
+    let connector_name = connector_label
+        .split('_') //connector_name will be the first string after splitting connector_label
+        .next()
+        .ok_or(errors::ApiErrorResponse::InternalServerError)?;
```

Comment:
can you add an attach printable here too? Since this variable is passed from the user, we need to handle all the cases